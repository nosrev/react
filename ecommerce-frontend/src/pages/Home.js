import React, { useEffect } from "react";
import HomeSlider from '../components/HomeSlider';
import Product from '../components/Product';
import Brands from '../components/Brands';
import { ReactComponent as ArrowRight } from '../svg/arrow-right.svg';
import halbanner1 from '../img/halbanner1.jpg';
import halbanner2 from '../img/halbanner2.jpg';
import { Link } from 'react-router-dom';
import ScrollIntoView from "../containers/ScrollIntoView";

const Home = () => {
    useEffect(() => {
        document.body.className = 'home';
        return () => { document.body.className = ''; }
    });

    const numbers = [...Array(8).keys()];

    return (
        <ScrollIntoView>
            <div>
                <HomeSlider />
                <div id="boxes" className="flex justify-between py-9 border border-slate-300 bg-white">
                    <div className="flex items-center justify-center flex-1 py-3">
                        <i className="icon-paper-plane text-3xl text-slate-300 mr-3"></i>
                        <div>
                            <p className="text-black text-lg font-semibold">Frete Grátis</p>
                            <span className="text-slate-400 text-sm">Compras acima de R$100</span>
                        </div>
                    </div>
                    <div className="flex items-center justify-center flex-1 py-3">
                        <i className="icon-refresh text-3xl text-slate-300 mr-3"></i>
                        <div>
                            <p className="text-black text-lg font-semibold">30 dias para troca</p>
                            <span className="text-slate-400 text-sm">Se tiver algum problema</span>
                        </div>
                    </div>
                    <div className="flex items-center justify-center flex-1 py-3">
                        <i className="icon-credit-card text-3xl text-slate-300 mr-3"></i>
                        <div>
                            <p className="text-black text-lg font-semibold">Seguro</p>
                            <span className="text-slate-400 text-sm">Pagamento 100% seguro</span>
                        </div>
                    </div>
                    <div className="flex items-center justify-center flex-1 py-3">
                        <i className="icon-user text-3xl text-slate-300 mr-3"></i>
                        <div>
                            <p className="text-black text-lg font-semibold">Suporte 24h</p>
                            <span className="text-slate-400 text-sm">Suporte dedicado</span>
                        </div>
                    </div>
                </div>
                <div className="container mx-auto">
                    <div className="flex justify-between space-x-3 top-banners">
                        <div className="basis-1/4 popular-products">
                            <div className="info">
                                <span className="icon-fire"></span>
                                <p className="title">
                                    Produto Popular
                                </p>
                                <Link to='/shop'>
                                    Mais Produtos
                                    <ArrowRight className="icon fill-white" />
                                </Link>
                            </div>
                        </div>
                        <div className="banner-large basis-3/4">
                            <span className="banner-badge text-primary bg-white">novo</span>
                            <p className="banner-subtitle text-white">lifestyle</p>
                            <p className="banner-title text-white">Algodão listrado</p>
                            <a href="/" className="banner-btn bg-white text-primary">R$250.00 <span className="border-l-2 border-primary">COMPRAR</span></a>
                        </div>
                    </div>

                    <div className="products flex flex-wrap">
                        {numbers.map(function(object, i){
                            return <div key={i} className="basis-2/4 md:basis-1/4"><Product type="home" /></div>
                        })}
                    </div>
                </div>

                <div className="box-fullwidth flex justify-center items-center">
                    <p className="text-white">Coleçoes incríveis especialmente pra você</p>
                </div>

                <div className="container mx-auto">
                    <div className="half-banners flex justify-between space-x-3">
                        <div className="half-banner">
                            <img src={halbanner1} alt="Príncipe de Gales Slim Fit" />
                            <p className="cat text-white">Ternos</p>
                            <p className="name text-white">Príncipe de Gales<br /> Slim Fit</p>
                            <p className="banner-price text-white">• R$ 2295.00</p>
                        </div>
                        <div className="half-banner">
                            <img src={halbanner2} alt="Cachecol caxemira de lã xadrez" />
                            <p className="cat text-white">Cachecóis</p>
                            <p className="name text-white">Cachecol caxemira<br /> de lã xadrez</p>
                            <p className="banner-price text-white">• R$ 425.00</p>
                        </div>
                    </div>

                    <Brands />
                </div>
            </div>
        </ScrollIntoView>
    );
};

export default Home;
