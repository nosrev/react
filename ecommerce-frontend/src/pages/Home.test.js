import renderer from 'react-test-renderer';
import Home from './Home';
import { MemoryRouter as Router } from 'react-router-dom';

it('render Home', () => {
  global.scrollTo = jest.fn();
  const component = renderer.create(
    <Router>
        <Home />
    </Router>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
