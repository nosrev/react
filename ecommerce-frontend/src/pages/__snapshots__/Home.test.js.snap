// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Home 1`] = `
<div>
  <div
    id="slider-wrapper"
  >
    <div
      className="slider"
    >
      <div
        className="slick-slider slick-initialized"
        dir="ltr"
      >
        <div
          className="slick-arrow slick-prev go-previous"
          onClick={[Function]}
          style={
            Object {
              "display": "block",
            }
          }
        >
          <svg
            className="icon w-8 fill-white"
          >
            arrow-left.svg
          </svg>
        </div>
        <div
          className="slick-list"
          onClick={[Function]}
          onKeyDown={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={null}
          onMouseMove={null}
          onMouseUp={[Function]}
          onTouchCancel={null}
          onTouchEnd={[Function]}
          onTouchMove={null}
          onTouchStart={[Function]}
          style={Object {}}
        >
          <div
            className="slick-track"
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            onMouseOver={[Function]}
            style={
              Object {
                "opacity": 1,
              }
            }
          >
            <div
              aria-hidden={false}
              className="slick-slide slick-active slick-current"
              data-index={0}
              onClick={[Function]}
              style={
                Object {
                  "left": -0,
                  "opacity": 1,
                  "outline": "none",
                  "position": "relative",
                  "transition": "opacity 200ms ease, visibility 200ms ease",
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <div
                  className="bg-black"
                  style={
                    Object {
                      "display": "inline-block",
                      "width": "100%",
                    }
                  }
                  tabIndex={-1}
                >
                  <img
                    alt="Fashion"
                    className="slide-img"
                    src="slide1.png"
                  />
                  <p
                    className="title text-blue2"
                  >
                    Fashion
                  </p>
                  <p
                    className="subtitle text-white"
                  >
                    Lançamentos
                  </p>
                  <a
                    className="btn-view inline-block bg-primary text-white border-2 border-primary hover:bg-white hover:text-primary"
                    href="/"
                  >
                    Comprar
                  </a>
                </div>
              </div>
            </div>
            <div
              aria-hidden={true}
              className="slick-slide"
              data-index={1}
              onClick={[Function]}
              style={
                Object {
                  "left": -0,
                  "opacity": 0,
                  "outline": "none",
                  "position": "relative",
                  "transition": "opacity 200ms ease, visibility 200ms ease",
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <div
                  className="bg-primary"
                  style={
                    Object {
                      "display": "inline-block",
                      "width": "100%",
                    }
                  }
                  tabIndex={-1}
                >
                  <img
                    alt="Casual"
                    className="slide-img"
                    src="slide1.png"
                  />
                  <p
                    className="title text-blue2"
                  >
                    Casual
                  </p>
                  <p
                    className="subtitle text-white"
                  >
                    Promoções
                  </p>
                  <a
                    className="btn-view inline-block bg-primary text-white border-2 border-primary hover:bg-white hover:text-primary"
                    href="/"
                  >
                    Comprar
                  </a>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          className="slick-arrow slick-next go-next"
          onClick={[Function]}
          style={
            Object {
              "display": "block",
            }
          }
        >
          <svg
            className="icon w-8 fill-white"
          >
            arrow-right.svg
          </svg>
        </div>
      </div>
    </div>
  </div>
  <div
    className="flex justify-between py-9 border border-slate-300 bg-white"
    id="boxes"
  >
    <div
      className="flex items-center justify-center flex-1 py-3"
    >
      <i
        className="icon-paper-plane text-3xl text-slate-300 mr-3"
      />
      <div>
        <p
          className="text-black text-lg font-semibold"
        >
          Frete Grátis
        </p>
        <span
          className="text-slate-400 text-sm"
        >
          Compras acima de R$100
        </span>
      </div>
    </div>
    <div
      className="flex items-center justify-center flex-1 py-3"
    >
      <i
        className="icon-refresh text-3xl text-slate-300 mr-3"
      />
      <div>
        <p
          className="text-black text-lg font-semibold"
        >
          30 dias para troca
        </p>
        <span
          className="text-slate-400 text-sm"
        >
          Se tiver algum problema
        </span>
      </div>
    </div>
    <div
      className="flex items-center justify-center flex-1 py-3"
    >
      <i
        className="icon-credit-card text-3xl text-slate-300 mr-3"
      />
      <div>
        <p
          className="text-black text-lg font-semibold"
        >
          Seguro
        </p>
        <span
          className="text-slate-400 text-sm"
        >
          Pagamento 100% seguro
        </span>
      </div>
    </div>
    <div
      className="flex items-center justify-center flex-1 py-3"
    >
      <i
        className="icon-user text-3xl text-slate-300 mr-3"
      />
      <div>
        <p
          className="text-black text-lg font-semibold"
        >
          Suporte 24h
        </p>
        <span
          className="text-slate-400 text-sm"
        >
          Suporte dedicado
        </span>
      </div>
    </div>
  </div>
  <div
    className="container mx-auto"
  >
    <div
      className="flex justify-between space-x-3 top-banners"
    >
      <div
        className="basis-1/4 popular-products"
      >
        <div
          className="info"
        >
          <span
            className="icon-fire"
          />
          <p
            className="title"
          >
            Produto Popular
          </p>
          <a
            href="/shop"
            onClick={[Function]}
          >
            Mais Produtos
            <svg
              className="icon fill-white"
            >
              arrow-right.svg
            </svg>
          </a>
        </div>
      </div>
      <div
        className="banner-large basis-3/4"
      >
        <span
          className="banner-badge text-primary bg-white"
        >
          novo
        </span>
        <p
          className="banner-subtitle text-white"
        >
          lifestyle
        </p>
        <p
          className="banner-title text-white"
        >
          Algodão listrado
        </p>
        <a
          className="banner-btn bg-white text-primary"
          href="/"
        >
          R$250.00 
          <span
            className="border-l-2 border-primary"
          >
            COMPRAR
          </span>
        </a>
      </div>
    </div>
    <div
      className="products flex flex-wrap"
    >
      <div
        className="basis-2/4 md:basis-1/4"
      >
        <a
          href="/product"
          onClick={[Function]}
        >
          <div
            className="product border border-slate-300"
          >
            <span
              className="product-badge"
            >
              oferta
            </span>
            <img
              alt="Camiseta algodão"
              src="produto.png"
            />
            <p
              className="product-name"
            >
              Camiseta algodão
            </p>
            <p
              className="product-price"
            >
              <span
                className="old-price"
              >
                R$200.00
              </span>
              <span
                className="special-price"
              >
                R$100.00
              </span>
            </p>
          </div>
        </a>
      </div>
      <div
        className="basis-2/4 md:basis-1/4"
      >
        <a
          href="/product"
          onClick={[Function]}
        >
          <div
            className="product border border-slate-300"
          >
            <span
              className="product-badge"
            >
              oferta
            </span>
            <img
              alt="Camiseta algodão"
              src="produto.png"
            />
            <p
              className="product-name"
            >
              Camiseta algodão
            </p>
            <p
              className="product-price"
            >
              <span
                className="old-price"
              >
                R$200.00
              </span>
              <span
                className="special-price"
              >
                R$100.00
              </span>
            </p>
          </div>
        </a>
      </div>
      <div
        className="basis-2/4 md:basis-1/4"
      >
        <a
          href="/product"
          onClick={[Function]}
        >
          <div
            className="product border border-slate-300"
          >
            <span
              className="product-badge"
            >
              oferta
            </span>
            <img
              alt="Camiseta algodão"
              src="produto.png"
            />
            <p
              className="product-name"
            >
              Camiseta algodão
            </p>
            <p
              className="product-price"
            >
              <span
                className="old-price"
              >
                R$200.00
              </span>
              <span
                className="special-price"
              >
                R$100.00
              </span>
            </p>
          </div>
        </a>
      </div>
      <div
        className="basis-2/4 md:basis-1/4"
      >
        <a
          href="/product"
          onClick={[Function]}
        >
          <div
            className="product border border-slate-300"
          >
            <span
              className="product-badge"
            >
              oferta
            </span>
            <img
              alt="Camiseta algodão"
              src="produto.png"
            />
            <p
              className="product-name"
            >
              Camiseta algodão
            </p>
            <p
              className="product-price"
            >
              <span
                className="old-price"
              >
                R$200.00
              </span>
              <span
                className="special-price"
              >
                R$100.00
              </span>
            </p>
          </div>
        </a>
      </div>
      <div
        className="basis-2/4 md:basis-1/4"
      >
        <a
          href="/product"
          onClick={[Function]}
        >
          <div
            className="product border border-slate-300"
          >
            <span
              className="product-badge"
            >
              oferta
            </span>
            <img
              alt="Camiseta algodão"
              src="produto.png"
            />
            <p
              className="product-name"
            >
              Camiseta algodão
            </p>
            <p
              className="product-price"
            >
              <span
                className="old-price"
              >
                R$200.00
              </span>
              <span
                className="special-price"
              >
                R$100.00
              </span>
            </p>
          </div>
        </a>
      </div>
      <div
        className="basis-2/4 md:basis-1/4"
      >
        <a
          href="/product"
          onClick={[Function]}
        >
          <div
            className="product border border-slate-300"
          >
            <span
              className="product-badge"
            >
              oferta
            </span>
            <img
              alt="Camiseta algodão"
              src="produto.png"
            />
            <p
              className="product-name"
            >
              Camiseta algodão
            </p>
            <p
              className="product-price"
            >
              <span
                className="old-price"
              >
                R$200.00
              </span>
              <span
                className="special-price"
              >
                R$100.00
              </span>
            </p>
          </div>
        </a>
      </div>
      <div
        className="basis-2/4 md:basis-1/4"
      >
        <a
          href="/product"
          onClick={[Function]}
        >
          <div
            className="product border border-slate-300"
          >
            <span
              className="product-badge"
            >
              oferta
            </span>
            <img
              alt="Camiseta algodão"
              src="produto.png"
            />
            <p
              className="product-name"
            >
              Camiseta algodão
            </p>
            <p
              className="product-price"
            >
              <span
                className="old-price"
              >
                R$200.00
              </span>
              <span
                className="special-price"
              >
                R$100.00
              </span>
            </p>
          </div>
        </a>
      </div>
      <div
        className="basis-2/4 md:basis-1/4"
      >
        <a
          href="/product"
          onClick={[Function]}
        >
          <div
            className="product border border-slate-300"
          >
            <span
              className="product-badge"
            >
              oferta
            </span>
            <img
              alt="Camiseta algodão"
              src="produto.png"
            />
            <p
              className="product-name"
            >
              Camiseta algodão
            </p>
            <p
              className="product-price"
            >
              <span
                className="old-price"
              >
                R$200.00
              </span>
              <span
                className="special-price"
              >
                R$100.00
              </span>
            </p>
          </div>
        </a>
      </div>
    </div>
  </div>
  <div
    className="box-fullwidth flex justify-center items-center"
  >
    <p
      className="text-white"
    >
      Coleçoes incríveis especialmente pra você
    </p>
  </div>
  <div
    className="container mx-auto"
  >
    <div
      className="half-banners flex justify-between space-x-3"
    >
      <div
        className="half-banner"
      >
        <img
          alt="Príncipe de Gales Slim Fit"
          src="halbanner1.jpg"
        />
        <p
          className="cat text-white"
        >
          Ternos
        </p>
        <p
          className="name text-white"
        >
          Príncipe de Gales
          <br />
           Slim Fit
        </p>
        <p
          className="banner-price text-white"
        >
          • R$ 2295.00
        </p>
      </div>
      <div
        className="half-banner"
      >
        <img
          alt="Cachecol caxemira de lã xadrez"
          src="halbanner2.jpg"
        />
        <p
          className="cat text-white"
        >
          Cachecóis
        </p>
        <p
          className="name text-white"
        >
          Cachecol caxemira
          <br />
           de lã xadrez
        </p>
        <p
          className="banner-price text-white"
        >
          • R$ 425.00
        </p>
      </div>
    </div>
    <div
      className="brands-wrapper"
    >
      <div
        className="slick-slider brands slick-initialized"
        dir="ltr"
      >
        <div
          className="slick-arrow slick-prev"
          onClick={[Function]}
          style={
            Object {
              "display": "block",
            }
          }
        >
          <svg
            className="icon w-8 left"
          >
            left.svg
          </svg>
        </div>
        <div
          className="slick-list"
          onClick={[Function]}
          onKeyDown={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={null}
          onMouseMove={null}
          onMouseUp={[Function]}
          onTouchCancel={null}
          onTouchEnd={[Function]}
          onTouchMove={null}
          onTouchStart={[Function]}
          style={Object {}}
        >
          <div
            className="slick-track"
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            onMouseOver={[Function]}
            style={
              Object {
                "WebkitTransform": "translate3d(0px, 0px, 0px)",
                "WebkitTransition": "",
                "msTransform": "translateX(0px)",
                "opacity": 1,
                "transform": "translate3d(0px, 0px, 0px)",
                "transition": "",
              }
            }
          >
            <div
              aria-hidden={true}
              className="slick-slide slick-cloned"
              data-index={-4}
              onClick={[Function]}
              style={
                Object {
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <img
                  alt="brand4"
                  src="brand4.png"
                  style={
                    Object {
                      "display": "inline-block",
                      "width": "100%",
                    }
                  }
                  tabIndex={-1}
                />
              </div>
            </div>
            <div
              aria-hidden={true}
              className="slick-slide slick-cloned"
              data-index={-3}
              onClick={[Function]}
              style={
                Object {
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <img
                  alt="brand2"
                  src="brand2.png"
                  style={
                    Object {
                      "display": "inline-block",
                      "width": "100%",
                    }
                  }
                  tabIndex={-1}
                />
              </div>
            </div>
            <div
              aria-hidden={true}
              className="slick-slide slick-cloned"
              data-index={-2}
              onClick={[Function]}
              style={
                Object {
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <img
                  alt="brand1"
                  src="brand1.png"
                  style={
                    Object {
                      "display": "inline-block",
                      "width": "100%",
                    }
                  }
                  tabIndex={-1}
                />
              </div>
            </div>
            <div
              aria-hidden={true}
              className="slick-slide slick-cloned"
              data-index={-1}
              onClick={[Function]}
              style={
                Object {
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <img
                  alt="brand3"
                  src="brand3.png"
                  style={
                    Object {
                      "display": "inline-block",
                      "width": "100%",
                    }
                  }
                  tabIndex={-1}
                />
              </div>
            </div>
            <div
              aria-hidden={false}
              className="slick-slide slick-active slick-current"
              data-index={0}
              onClick={[Function]}
              style={
                Object {
                  "outline": "none",
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <img
                  alt="brand1"
                  src="brand1.png"
                  style={
                    Object {
                      "display": "inline-block",
                      "width": "100%",
                    }
                  }
                  tabIndex={-1}
                />
              </div>
            </div>
            <div
              aria-hidden={false}
              className="slick-slide slick-active"
              data-index={1}
              onClick={[Function]}
              style={
                Object {
                  "outline": "none",
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <img
                  alt="brand2"
                  src="brand2.png"
                  style={
                    Object {
                      "display": "inline-block",
                      "width": "100%",
                    }
                  }
                  tabIndex={-1}
                />
              </div>
            </div>
            <div
              aria-hidden={false}
              className="slick-slide slick-active"
              data-index={2}
              onClick={[Function]}
              style={
                Object {
                  "outline": "none",
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <img
                  alt="brand3"
                  src="brand3.png"
                  style={
                    Object {
                      "display": "inline-block",
                      "width": "100%",
                    }
                  }
                  tabIndex={-1}
                />
              </div>
            </div>
            <div
              aria-hidden={false}
              className="slick-slide slick-active"
              data-index={3}
              onClick={[Function]}
              style={
                Object {
                  "outline": "none",
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <img
                  alt="brand4"
                  src="brand4.png"
                  style={
                    Object {
                      "display": "inline-block",
                      "width": "100%",
                    }
                  }
                  tabIndex={-1}
                />
              </div>
            </div>
            <div
              aria-hidden={true}
              className="slick-slide"
              data-index={4}
              onClick={[Function]}
              style={
                Object {
                  "outline": "none",
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <img
                  alt="brand2"
                  src="brand2.png"
                  style={
                    Object {
                      "display": "inline-block",
                      "width": "100%",
                    }
                  }
                  tabIndex={-1}
                />
              </div>
            </div>
            <div
              aria-hidden={true}
              className="slick-slide"
              data-index={5}
              onClick={[Function]}
              style={
                Object {
                  "outline": "none",
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <img
                  alt="brand1"
                  src="brand1.png"
                  style={
                    Object {
                      "display": "inline-block",
                      "width": "100%",
                    }
                  }
                  tabIndex={-1}
                />
              </div>
            </div>
            <div
              aria-hidden={true}
              className="slick-slide"
              data-index={6}
              onClick={[Function]}
              style={
                Object {
                  "outline": "none",
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <img
                  alt="brand3"
                  src="brand3.png"
                  style={
                    Object {
                      "display": "inline-block",
                      "width": "100%",
                    }
                  }
                  tabIndex={-1}
                />
              </div>
            </div>
            <div
              aria-hidden={true}
              className="slick-slide slick-cloned"
              data-index={7}
              onClick={[Function]}
              style={
                Object {
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <img
                  alt="brand1"
                  src="brand1.png"
                  style={
                    Object {
                      "display": "inline-block",
                      "width": "100%",
                    }
                  }
                  tabIndex={-1}
                />
              </div>
            </div>
            <div
              aria-hidden={true}
              className="slick-slide slick-cloned"
              data-index={8}
              onClick={[Function]}
              style={
                Object {
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <img
                  alt="brand2"
                  src="brand2.png"
                  style={
                    Object {
                      "display": "inline-block",
                      "width": "100%",
                    }
                  }
                  tabIndex={-1}
                />
              </div>
            </div>
            <div
              aria-hidden={true}
              className="slick-slide slick-cloned"
              data-index={9}
              onClick={[Function]}
              style={
                Object {
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <img
                  alt="brand3"
                  src="brand3.png"
                  style={
                    Object {
                      "display": "inline-block",
                      "width": "100%",
                    }
                  }
                  tabIndex={-1}
                />
              </div>
            </div>
            <div
              aria-hidden={true}
              className="slick-slide slick-cloned"
              data-index={10}
              onClick={[Function]}
              style={
                Object {
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <img
                  alt="brand4"
                  src="brand4.png"
                  style={
                    Object {
                      "display": "inline-block",
                      "width": "100%",
                    }
                  }
                  tabIndex={-1}
                />
              </div>
            </div>
            <div
              aria-hidden={true}
              className="slick-slide slick-cloned"
              data-index={11}
              onClick={[Function]}
              style={
                Object {
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <img
                  alt="brand2"
                  src="brand2.png"
                  style={
                    Object {
                      "display": "inline-block",
                      "width": "100%",
                    }
                  }
                  tabIndex={-1}
                />
              </div>
            </div>
            <div
              aria-hidden={true}
              className="slick-slide slick-cloned"
              data-index={12}
              onClick={[Function]}
              style={
                Object {
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <img
                  alt="brand1"
                  src="brand1.png"
                  style={
                    Object {
                      "display": "inline-block",
                      "width": "100%",
                    }
                  }
                  tabIndex={-1}
                />
              </div>
            </div>
            <div
              aria-hidden={true}
              className="slick-slide slick-cloned"
              data-index={13}
              onClick={[Function]}
              style={
                Object {
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <img
                  alt="brand3"
                  src="brand3.png"
                  style={
                    Object {
                      "display": "inline-block",
                      "width": "100%",
                    }
                  }
                  tabIndex={-1}
                />
              </div>
            </div>
          </div>
        </div>
        <div
          className="slick-arrow slick-next"
          onClick={[Function]}
          style={
            Object {
              "display": "block",
            }
          }
        >
          <svg
            className="icon w-8 right"
          >
            right.svg
          </svg>
        </div>
      </div>
    </div>
  </div>
</div>
`;
